cluster-add-user/cluster-add-user.sh

```diff
@@ -57,7 +57,8 @@ sudo -u $username cp ../slurm-examples/* /home/$username/shared/
 
 gecos=$(getent passwd $username | cut -d ':' -f 5)
 
-cat <<HERE > ~/shared/remote-add-user
+remoteFile=~/shared/remote-add-user
```

Comment:
Tips: I think `~` is `home/username`, and quotes will cause  [Tilde Expansion](https://www.gnu.org/software/bash/manual/bash.html#Tilde-Expansion) invalid. 

For variable uses, I do not find errors.